src/com/luxoft/java/course/shevtsov/homework/myUtilities/MyUtilities.java

```diff
@@ -0,0 +1,30 @@
+package com.luxoft.java.course.shevtsov.homework.myUtilities;
+
+import java.util.Scanner;
+
+public class MyUtilities {
+    private static Scanner sc = new Scanner(System.in);
+
+    public static int inputInt() {
+        String input;
+
+        while (true) {
+            System.out.print("Input integer number: ");
+            input = sc.nextLine();
+            try {
+                return Integer.parseInt(input);
+            } catch (NumberFormatException numFormEx) {
+                System.out.println("Invalid input. Please repeat.");
+            }
+        }
+    }
+
+    public static boolean ifArrSotredByAsc(int[] ints) {
```

Comment:
Better use is, because you return result of this condition

Reply:
Why asc?

Reply:
'Better use is, because you return result of this condition' - done
asc from ascending. isArrSotredByAscending would be better? or smth else?